### PR TITLE
CTECH-2595 Added notebook for T09002

### PR DESCRIPTION
**Pull request by MikeMcGarry**
# Pull Request Checklist

- [x] Changes follow the [style guide](https://github.com/finbourne/sample-notebooks/blob/master/docs/FINBOURNE%20notebook%20style%20guide.ipynb)
- [ ] Tests pass
- [x] Raised the PR against the `develop` branch

# Description of the PR

Describe the code changes for the reviewers, explain the solution you have provided and how it fixes the issue

### CODE REVIEW

**Comment by MikeMcGarry** [comment]
Notebook is not idempotent, will update to be idempotent.

**Review by rizwansaeed — DISMISSED** [review]

File: university/T09002-EventsAndNotifications-SubscribingToEvents/T09002-EventsAndNotifications-SubscribingToEvents.ipynb [hunk 1]
```diff
@@ -0,0 +1,516 @@
+{
```
Comment (rizwansaeed) [inline university/T09002-EventsAndNotifications-SubscribingToEvents/T09002-EventsAndNotifications-SubscribingToEvents.ipynb:1]: <p>were you going to put the check and create in different cells (for the slides) or was that not needed any more?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/finbourne/sample-notebooks/pull/617/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

**Pull request closed without merging**